crt_portal/cts_forms/templates/vot_resources.html

```diff
@@ -0,0 +1,111 @@
+{% extends "landing.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block meta_title %}
+  {% trans "Civil Rights Division - Department of Justice - Voting resources" as meta_title %}
+  <meta property="og:title" content="{{meta_title}}" />
+  <meta property="name" content="{{meta_title}}" />
+{% endblock meta_title %}
+
+{% block page_title %}
+  <title>
+    {% trans "Civil Rights Division - Department of Justice - Voting resources" %}
+  </title>
+{% endblock %}
+
+{% block content %}
+<div class="crt-landing--section crt-landing--pale crt-subpage">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+        <ul class="usa-sidenav crt-subpage--nav" id="toc">
+          <li class="usa-sidenav__item">
+            <a href="#get-help">{% trans "How Do I..." %}</a>
+          </li>
+          <ul class="sublist">
+            <li class="usa-sidenav__item">
+              <a href="#criminal-violation">{% trans "Report a possible federal criminal violation" %}</a>
+            </li>
+            <li class="usa-sidenav__item">
+              <a href="#polling-place">{% trans "Find polling place location and hours" %}</a>
+            </li>
+            <li class="usa-sidenav__item">
+              <a href="#civilrights-violation">{% trans "Report a possible civil rights violation" %}</a>
+            </li>
+            <li class="usa-sidenav__item">
+              <a href="#voting-rights">{% trans "Learn about my voting rights" %}</a>
+            </li>
+          </ul>
+        </ul>
+      </div>
+      <div class="tablet:grid-col-8">
+        <h1 id="get-help">
+          {% trans "Report voting issues" %}
+        </h1>
+        <div class="crt-landing--separator"></div>
+
+        <section id="criminal-violation">
+          <h2>{% trans "Report a possible federal criminal violation" %}</h2>
+          
+          <p>
+            {% trans "To report a possible federal crime, go to" %} <a class="external-link--blue" href="https://tips.fbi.gov">tips.fbi.gov</a>  {% trans " or call 800-CALL-FBI." %}
+          </p>
+          <p>
+            {% trans "EXAMPLES" %}
+          </p>
+          <ul>
+            <li>{% trans "Threat against voters" %}</li>
+            <li>{% trans "Threat against election officials" %}</li>
+            <li>{% trans "Election fraud" %}</li>
+          </ul>
+        </section>
+        <section id="polling-place">
+          <h2>{% trans "Find polling place location and hours" %}</h2>
+
+          <p>
+            {% trans "Go to" %} <a class="external-link--blue" href="https://usa.gov/election-office">usa.gov/election-office</a>  {% trans " to find your state election website." %}
```

Comment:
Does this need an external link modal?

Reply:
this is external to DOJ, but still federal site, so no modal.